samples/bmi/calculator/src/main/java/io/servicecomb/samples/bmi/CalculatorServiceImpl.java

```diff
@@ -30,16 +38,34 @@
    * {@inheritDoc}
    */
   @Override
-  public double calculate(double height, double weight) {
+  public Map<String, String> calculate(double height, double weight) {
     if (height <= 0 || weight <= 0) {
       throw new IllegalArgumentException("Arguments must be above 0");
     }
     double heightInMeter = height / 100;
     double bmi = weight / (heightInMeter * heightInMeter);
-    return roundToOnePrecision(bmi);
+    
```

Comment:
Please create another interface to get the system information instead of hacking the old interface.  BTW, it's not a good practice that using map for response message.